packages/devtools_app/lib/src/screens/vm_developer/object_inspector/class_hierarchy_explorer_controller.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+import 'package:flutter/foundation.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../shared/globals.dart';
+import '../../../shared/primitives/trees.dart';
+
+class ClassHierarchyExplorerController {
+  ValueListenable<List<ClassHierarchyNode>> get selectedIsolateClassHierarchy =>
+      _selectedIsolateClassHierarchy;
+  final _selectedIsolateClassHierarchy =
+      ValueNotifier<List<ClassHierarchyNode>>([]);
```

Comment:
does this need to be a list? looks like there is only ever 1 or 0 elements in it